src/types/model.ts

```diff
@@ -132,9 +132,12 @@ export class Registry {
     genesisBlockNumber: number
   ): Promise<Registry> {
     const db = storage.getDB();
-    const str = await db.get(
-      getNetworkStorageKey(CONDITIONAL_ORDER_REGISTRY_STORAGE_KEY, network)
-    );
+    const str = await db
```

Comment:
not from this PR, but str is a not so good name for a var

Reply:
Will defer handling this to #65 as there are changes here that will result in more merge conflicts.